### PR TITLE
Multimer

### PR DESCRIPTION
**Pull request by Fazziekey**
## What does this PR do?

- fix the templatepairstackblock

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [x] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

In short, see the following bullet-list:

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by popfido** [comment]
LGTM. Could you briefly describe why the order of these codes in template pair stack block is wrong? Maybe where does the the original paper describe it?

**Comment by Fazziekey** [comment]
> 
I think these code is meaningless
EvLAQnUtQP

**Comment by Fazziekey** [comment]
this  is the origin code for multimer
image

**Comment by popfido** [comment]
@ycsos is more familiar with this module. I think he may give you a better explain & review

**Comment by popfido** [comment]
I have confirmed that this change has already accepted by our private version, LGTM

**Review by popfido — APPROVED** [review]
